### PR TITLE
Working on using the Google Maps API

### PR DESCRIPTION
**Pull request by ruchiishahh**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -34,14 +34,15 @@ <h2>
...
       <div id="input-container">
         <form action="/data" method="POST"> 
             <h2> Enter a basketball player here: </h2>
-            <input type="text" name="text-input" minlength="1" maxlength="50" title="50 characters minimum" required>
+            <input type="text" name="text-input" minlength="1" maxlength="50" title="50 characters minimum" id="user-input-text" required>
             <input onchange="getDataComment()" type="number" id="numComments" name="numComments" min="1" max="10" value=5>
-            <input type="submit" value="Submit!"/>
-            </br>
+            <input id="submit-dataComments" type="submit" value="Submit!"/>
+            </br></br>
```
Comment (chaconnewu) [inline portfolio/src/main/webapp/index.html:45]: you can do `<br /> `

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -192,6 +243,27 @@ p.social-media-text{
     top:1px;
 }
 
+#submit-dataComments{
```
Comment (chaconnewu) [inline portfolio/src/main/webapp/style.css:246]: indentations are not aligned for the css changes here.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -31,12 +31,71 @@ function getDataComment() {
...
+            miniP.innerText = dataComment.message + " | Author: " + dataComment.creator;
             commentContainer.appendChild(miniP);
         })
     });
 }
 
+/**
+ * Animates the markers onto the map
+ */
+function toggleBounce() {
```
Comment (chaconnewu) [inline portfolio/src/main/webapp/script.js:43]: Change all "var" to "const" (or "let" if the value needs to change)

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (ruchiishahh) [reply portfolio/src/main/webapp/index.html]: Thanks, I'll use that instead!

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (ruchiishahh) [reply portfolio/src/main/webapp/style.css:242]: Fixed, thanks!

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (ruchiishahh) [reply portfolio/src/main/webapp/script.js:43]: Fixed, thanks!

**Pull request merged**